src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -21,6 +22,8 @@
 
 import javax.annotation.Nonnull;
 
+import static org.bukkit.Bukkit.getServer;
```

Comment:
Why the static import here...

Reply:
Im not really sure how these work, It auto imported when I used getServer, and getting rid of static breaks stuff... Could u explain pls

Reply:
Bukkit.getServer

You're calling a method from the Bukkit class.